src/main/java/org/jboss/tools/intellij/openshift/actions/component/ShowLogComponentAction.java

```diff
@@ -123,11 +123,19 @@ private  boolean isDebug(Component component) {
     }
 
     private  boolean isLogRunningForDevOrDebug(ComponentNode componentNode) throws IOException {
-        return componentNode.getRoot().getOdo().isLogRunning(componentNode.getComponent().getPath(), componentNode.getComponent().getName(), false);
+        Odo odo = componentNode.getRoot().getOdo().getNow(null);
```

Comment:
odo is already present before this call, better to pass it along. and it's also not null at this point ( from OdoAction )

Reply:
can do, I agree. But these methods are also called from #isVisible where odo isnt' present. So we need to add odo retrieval there.

Reply:
ok then, better to pass a not null odo to those methods.

Reply:
added the annotation on `OdoAction.actionPerformed` and all it's subclasses:

```java
public actionPerformed(AnActionEvent anActionEvent, Object selected, @NotNull Odo odo)
```